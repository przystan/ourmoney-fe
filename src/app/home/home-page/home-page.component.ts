import { Component, ChangeDetectionStrategy } from '@angular/core';

@Component({
  selector: 'app-home-page',
  template: `
    <p>
      home-page works!
    </p>
  `,
  styleUrls: ['./home-page.component.scss'],
  changeDetection: ChangeDetectionStrategy.OnPush
})
export class HomePageComponent {}
