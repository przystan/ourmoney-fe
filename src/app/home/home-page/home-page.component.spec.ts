import { Spectator, createComponentFactory } from '@ngneat/spectator';
import { HomePageComponent } from './home-page.component';

describe('HomePageComponent', () => {
  let spectator: Spectator<HomePageComponent>;

  const createComponent = createComponentFactory({
    component: HomePageComponent
  });

  beforeEach(() => {
    spectator = createComponent();
  });

  it('should match snapshot', () => {
    expect(spectator.fixture).toMatchSnapshot();
  });
});
