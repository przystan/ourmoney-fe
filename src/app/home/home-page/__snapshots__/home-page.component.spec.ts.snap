// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePageComponent should match snapshot 1`] = `
<app-home-page>
  <p>
     home-page works! 
  </p>
</app-home-page>
`;
